astro_2.0/features/CreateProposal/components/UpdateGroupContent/UpdateGroupContent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, useState } from 'react';
+import cn from 'classnames';
+import { useFormContext } from 'react-hook-form';
+
+import { TGroup } from 'types/dao';
+
+import { Button } from 'components/button/Button';
+import { Icon } from 'components/Icon';
+
+import styles from './UpdateGroupContent.module.scss';
+
+type Props = {
+  groups: TGroup[];
+  getDataFromContext?: boolean;
+};
+
+export const UpdateGroupContent: FC<Props> = ({
+  groups,
+  getDataFromContext = false,
+}) => {
+  const formContext = useFormContext();
+
+  let groupsList: TGroup[] = [];
+
+  if (getDataFromContext && formContext) {
+    const { watch } = formContext;
+
+    groupsList = watch('groups');
+  } else {
+    groupsList = groups;
+  }
+
+  const [activeGroupSlug, setActiveGroupSlug] = useState(groupsList[0].slug);
+
+  const activeGroup =
+    groupsList.find(group => group.slug === activeGroupSlug) || groupsList[0];
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.list}>
+        <h4 className={styles.listTitle}>Groups</h4>
```

Comment:
Can I ask you to mark files that require localization with the
```
// TODO requires localisation
```
comment at the top of the file, please?